src/components/HomePage/Banner.js

```diff
@@ -143,13 +149,15 @@ function Banner() {
 		completed: PropTypes.bool.isRequired
 	};
 
+	const smallScreen = useMediaQuery(theme.breakpoints.down('sm'));
+
 	return (
 		<div className={classes.container}>
```

Comment:
Idea: reduce the padding on the `.container` for smaller screen sizes. I think `padding: '12px 4px'` looks good.